src/components/scenes/WalletDetailsScene.tsx

```diff
@@ -328,36 +247,90 @@ function WalletDetailsComponent(props: Props) {
       renderFooter={renderFooter}
     >
       {({ insetStyle, undoInsetStyle }) => (
-        <View style={undoInsetStyle}>
-          <Animated.FlatList
-            style={styles.flatList}
-            ref={flashListRef}
-            contentContainerStyle={{
-              paddingTop: insetStyle.paddingTop + theme.rem(0.5),
-              paddingBottom: insetStyle.paddingBottom + theme.rem(0.5),
-              paddingLeft: insetStyle.paddingLeft + theme.rem(0.5),
-              paddingRight: insetStyle.paddingRight + theme.rem(0.5)
-            }}
-            data={listItems}
-            keyboardDismissMode="on-drag"
-            keyboardShouldPersistTaps="handled"
-            keyExtractor={keyExtractor}
-            ListEmptyComponent={emptyComponent}
-            ListHeaderComponent={topArea}
-            onEndReachedThreshold={0.5}
-            renderItem={renderItem}
-            onScroll={handleScroll}
-            scrollIndicatorInsets={SCROLL_INDICATOR_INSET_FIX}
-            // Android scroll gestures break without refreshControl given the
-            // combination of props we use on this Animated.FlatList.
-            refreshControl={refreshControl}
+        <Reanimated.ScrollView
```

Comment:
Nice! I was about to make this change too when I noticed this left over. Thanks for doing this.

However, why wouldn't we use the SceneWrapper's scroll view instead of a new `Reanimated.ScrollView`? That way we can remove all the "insetStyle" and "undoInsetStyle" work since we don't need a custom scroll component.

PS: In the future, I'd love the API to change to just implementing a scroll component where we need it instead of a `scroll` flag to the `SceneWrapper`. This way we can tuck away all the inset/undo-inset work into the custom scroll components we want to use. This would be a better DX, but we just haven't done the work yet.

Reply:
We need to set some props on the scroll view that we can't get to if we use the `SceneWrapper` one.

I agree that a `SceneScrollView` or such would be nice. It could use context extract the `insetStyle` from the `SceneWrapper` so the scene doesn't need to do anything fancy to hook them up. Then we could pass in the props we need, such as on this scene.

Reply:
BTW, I don't see a need for the search bar on the wallet details scene. There are only 5 transactions visible. Search isn't necessary

Reply:
I understand now after discussion. We want search because it's convenient. Though we can improve the search feature, these are reasonable fixes to a feature that may not be fully polished up. In the future we may want to replace the whole scene with a FlatList component just like the TransactionListScene has that is searchable and scrollable. This could be thought of as a "search modal view" that shows transactions (maybe other items) in the search list when search is activated.